commons/src/main/scala/com/lynbrookrobotics/potassium/commons/drivetrain/UnicycleDrive.scala

```diff
@@ -33,6 +33,7 @@ trait UnicycleHardware {
 
   val forwardPosition: Signal[Length]
   val turnPosition: Signal[Angle]
+  val position: Signal[Point]
```

Comment:
In general, I feel like you're overusing polling signals. I only added polling signals for the specific use case of gyros, where you want to treat a periodically updated value as a continuous stream. You really shouldn't use them whenever possible, though.